source/_integrations/ring.markdown

```diff
@@ -72,6 +92,133 @@ Two camera entities are provided: `live_view` and `last_recording`.
 Please note that downloading and playing Ring video from the `last_recording` camera will require a Ring Protect plan.
 {% endimportant %}
 
+### Event
+
+The event entity captures events like doorbell rings, motion alerts, and intercom unlocking.
+
+### Sensor
+
+Once you have enabled the [Ring integration](/integrations/ring), you can start using the sensor platform. Currently, it supports battery level and Wi-Fi signal.
+
+The volume sensors are being replaced with the number entity, which allows setting the volume. You should migrate any automations using the volume sensors to the number entity by release 2025.4.0.
+
+### Siren
+
+- Adds a siren entity for every camera that supports a siren. Note the siren will only turn on for 30 seconds before automatically turning off.
+- Adds a siren entity for chimes to play the test sound.
+
+### Switch
+
+Once you have enabled the [Ring integration](/integrations/ring), you can start using the switch platform.
+
+- Motion detection - Switches motion detection on and off for cameras.
+- In-home chime - Switches on and off a mechanical or digital chime connected to a ring doorbell.
+
+### Light
+
+Once you have enabled the [Ring integration](/integrations/ring), you can start using the light platform. This will add a light for every camera that supports a light (such as a floodlight).
+
+### Number
+
+Once you have enabled the [Ring integration](/integrations/ring), you can start using the number platform.
+Currently, it supports showing and setting the volume of the doorbell/chime ring, intercom voice volume, and intercom microphone volume.
+
+## Data updates
+
+The Ring cloud API is polled for data updates every 60 seconds. When you make changes through Home Assistant (e.g., switching motion detection on), the device's state is updated immediately rather than waiting for the next poll.
```

Comment:
```suggestion
The Ring cloud API is {% term polling polled %} for data updates every 60 seconds. When you make changes through Home Assistant (for example by switching motion detection on), the device's state is updated immediately rather than waiting for the next  {% term polling poll %}.
```